Added content to CI section on htmlproofer (#93)
pyOpenSci/handbook: community/github/continuous-integration.md

```

If all of the HTMLProofer checks pass, you won't see any errors. Instead, you will see a bright green checkmark. However, sometimes you may encounter an error. 
Below is an image the shows a CI error associated with the build:

```{figure} /images/github-images/htmlproofer-fail.png
:alt: A screenshot of the GitHub CI notifications for a sample pyOpenSci repository. There is a header that reads All checks have passed, with the text 3 failing checks beneath it. To the right side of the header is linked text reading Hide all checks. Below the header are three rows of information. The first row reads Build Push github pages / build-book (pull_request), the second reads ci/circleci: build_book -- Your tests failed on CircleCI!, and the third reads pre-commi.ci - pr -- checks completed with failures. There is a red check next to each line of text. Beneath these rows is the text This branch has no conflicts with the base branch. Merging can be performed automatically. There is also a green Merge pull request button.

A screenshot from a GitHub PR where all checks have passed.
```

When there is an error in the pyOpenSci CI build, there are a few things to consider. 

> 1. Are the CI errors are unrelated to your pull request? pyOpenSci's CI builds run on all of the files in the repository by default. In some cases, a link might become broken simply because of the website being down. And that break has nothing to do with your pull request! In that case pyOpenSci repository maintianers can still merge the pull request, even if there are failed checks. 

Reviewer: ```suggestion
1. Are the CI errors unrelated to your pull request? pyOpenSci's CI builds run on all the files in the repository by default. Sometimes, a link might become broken simply because the website is down. And that break has nothing to do with your pull request! In that case, pyOpenSci repository maintainers can still merge the pull request, even if there are failed checks. 
```

A screenshot of the build-book jobs page in GitHub with htmlproofer expanded.
```

In this case, HTMLProofer told you that none of the images submitted with this PR could be found. In this case you should go back and double check that the images are in the correct folder and that their paths' are correct. In this case, the author forgot the leading forward slash in the image paths.

If you see a broken HTMLProofer build, you can continue to make updates to the pull request either locally or through inline comments, until no
errors are returned by HTMLProofer. Once you have a green check next to the build, it's time to submit your pull request for review!

:::{note}
In some cases you may be unsure as to why HTMLProofer or any part of our pyOpenSci CI checks are failing. In these cases, feel free to ping a pyOpenSci 

> repository maintianer for help. We understand that Continuout Integration (CI) can be confusing to navigate even for seasoned GitHub users.

Reviewer: ```suggestion
repository maintainer for help. We understand that Continuous Integration (CI) can be confusing to navigate, even for seasoned GitHub users.
```
